Makefile

```diff
@@ -136,11 +136,9 @@ format:
 	test -z "$$(find . -path ./vendor -prune -type f -o -name '*.go' -exec gofmt -w {} + | tee /dev/stderr)"
 
 HAS_NPM          := $(shell command -v npm;)
```

Comment:
Can we remove npm from here also?

Reply:
Seems so to me, and the mention of npm in developers.md, too!